src/index.tsx

```diff
@@ -202,9 +202,16 @@ class PayPalButton extends React.Component<PayPalButtonProps, PayPalButtonState>
     }
 
     private addPaypalSdk() {
-        const { options, onButtonReady } = this.props;
+        const { onButtonReady } = this.props;
```

Comment:
Refactor this, like:
```
const separator = (key: string): string => key.split(/(?=[A-Z])/).join("-").toLowerCase();
        const createQueryParam = (object: object, modifier: Function): string => Object.entries(object)
            .reduce((acc: string [], [key, value]: [string, string]) => {
                acc.push(`${modifier(key)}=${value}`);
                return acc;
            }, []).join("&");
        const queryParam = createQueryParam({...PayPalButton.defaultProps.options, ...options}, separator);

        const script = document.createElement("script");
        script.type = "text/javascript";
        script.src = `https://www.paypal.com/sdk/js?${queryParam}`;
```

Reply:
fixed, can't update eslint right now (have some errors on Ubuntu with linter running)

Reply:
thanks!